pubspec.yaml

```diff
@@ -114,6 +121,10 @@ dependencies:
   share: 2.0.4
 
   # flutter_contacts
+  # ISSUE: https://github.com/linagora/tmail-flutter/issues/60
```

Comment:
Related to this library, the device contact in Android and iOS is complicated database, so we customized to optimizing the way to search contact focus to name and email only. We use it not only here but also in LinShare mobile.

Reply:
Awesome comment. Please state in the comment that LinShare app also use it, and please remove the unused TODOs...